src/core/ipv4/ip4_napt.c

```diff
@@ -86,47 +88,67 @@ struct portmap_table {
 };
 
 static u16_t napt_list = NO_IDX, napt_list_last = NO_IDX, napt_free = 0;
+static u8_t ip_portmap_max = 0;
 
-static struct napt_table *ip_napt_table = NULL;
-static struct portmap_table *ip_portmap_table = NULL;
+static struct ip_napt_entry *ip_napt_table = NULL;
+static struct ip_portmap_entry *ip_portmap_table = NULL;
 
-static int nr_active_napt_tcp = 0, nr_active_napt_udp = 0, nr_active_napt_icmp = 0;
-static uint16_t ip_napt_max = 0;
-static uint8_t ip_portmap_max = 0;
+static struct ip_napt_stats napt_stats;
+
+static void ip_napt_gc(uint32_t now, bool force);
+
+#define WRAPPED_AROUND(a, b) ((((a) ^ (b)) & (1UL << 31)) != 0)
 
 #if NAPT_DEBUG
 /* Print NAPT table using LWIP_DEBUGF
 */
-static void
+#define DPRINTF(m) LWIP_DEBUGF(NAPT_DEBUG, m)
+//#define DPRINTF(m) printf m
+void
```

Comment:
Could this still be static?
```suggestion
static void
```